pkg/network/tracer/tracer.go

```diff
@@ -818,15 +828,5 @@ func newUSMMonitor(c *config.Config, tracer connection.Tracer, bpfTelemetry *net
 		return nil
 	}
 
-	if c.EnableHTTP2Monitoring {
-		log.Info("http2 monitoring enabled")
-	}
-	if c.EnableKafkaMonitoring {
-		log.Info("kafka monitoring enabled")
-	}
-	if c.EnableGoTLSSupport {
```

Comment:
we should print the log later in case of failing fallback scenario
* gotls enabled
* usm.NewMonitor()
* core load failed (ex: no headers)
* runtime failed (ex: no kernel headers)
* GoTLSSupport is marked/log disabled [here](https://github.com/DataDog/datadog-agent/blob/main/pkg/network/tracer/tracer.go#L806)